### ISSUE TITLE
Link for FDOT information page incorrect

### ISSUE THREAD

**Issue by iannerney** [SkywayBridgeStatus/website#3]
The FDOT page for Skyway Bridge construction and closure information has changed, and should be updated on the homepage.

- Current URL: http://www.mytbi.com/projects/projects.asp?roadID=9
- New URL: http://www.fdottampabay.com/projects/road/sunshine-skyway

Additionally the title and alt properties are incorrect for the FDOT image, and should be updated.
`<img class="status-source__image" src="assets/img/fdot-logo.png" title="FL511 Traffic and Commuter Information" alt="FL511 logo" align="left"/>`

**Comment by iannerney** [comment]
After reviewing the data source I decided that I'm going to restructure this section of the home page, as the FDOT site only appears to report pending road construction projects. I've signed up for email notifications, and will post any planned closures in the `planned_closures` data table for reporting on the home page.

As a side note, when researching if this was the most valid link to post on the home page, I stumbled across a [similar but different page](http://www.fdottampabay.com/projects/road/skyway-bridge) for the Skyway, but there was nothing posted on this page. Not considering this page as valid.

I have a bit of tidying up to do, but a commit for this change is coming soon.
